assets/i18n/eu.json

```diff
@@ -131,27 +139,18 @@
 	"I18N_INTERACTIONS_SUBMIT": "Bidali",
 	"I18N_LANGUAGE_FOOTER_VIEW_IN": "Ikusi Oppia hemen:",
 	"I18N_LEARNER_DASHBOARD_COMPLETED_SECTION": "Bukatuta",
-	"I18N_LEARNER_DASHBOARD_EMPTY_FEEDBACK_THREADS": "Oraindik ez duzu iritzi hari aktiborik. Zure iritziak gure ikasgaien kalitatea hobetzen laguntzen du. Horretarako, gure ikasgaietako bat hasi eta zure iritzi baliotsua bidaliko dugu!",
 	"I18N_LEARNER_DASHBOARD_EXPLORATIONS_SORT_BY_LAST_PLAYED": "Azken erreprodukzioa",
-	"I18N_LEARNER_DASHBOARD_FEEDBACK_SECTION": "Kritiken Eguneraketa",
```

Comment:
This line has not been added to the new changes, can you please explain if it is intentional?

Reply:
Hi, @lkbhitesh07 Thanks for the review. Earlier, the Feedback Updates tab is present in the learner dashboard main menu and now in the PR, this has been shifted to the profile-pic drop-down menu.
The "I18N_TOPNAV_FEEDBACK_UPDATES" label has been used in all the files where I18N_LEARNER_DASHBOARD_FEEDBACK_SECTION was.
I have missed in some files. Now I have added  PTAL.
Thanks!